djangocms_text_ckeditor/fields.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, *args, **kwargs):
         self.configuration = kwargs.pop('configuration', None)
         super(HTMLField, self).__init__(*args, **kwargs)
 
-    def from_db_value(self, value, expression, connection, context):
+    def from_db_value(self, value, expression, connection):
```

Comment:
How about replacing with `*args, **kwargs` to keep compatibility with older django versions?
```suggestion
    def from_db_value(self, value, expression, connection, *args, **kwargs):
```

Reply:
@yakky I'm against adding unnecessary fields `*args` and `**kwargs` which never won't be filled. Instead I added a fallback to keep it compatible with Django-1.11.

BTW. This bug prevents me upgrading to Django-3.0. So please merge it and build a new release asap. If you want, I can do that with the appropriate permissions.